docs/concepts/tokens/decentralized-exchange/index.md

```diff
@@ -4,17 +4,41 @@ parent: tokens.html
 metadata:
   indexPage: true
 seo:
-    description: The XRP Ledger contains a fully functional exchange where users can trade tokens for XRP or each other.
+    title: Decentralized Exchange (DEX)
```

Comment:
If you're changing the title on line 12, you probably don't need to add a matching frontmatter attribute.